modules/health_quest/app/services/health_quest/patient_generated_data/fhir_client.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module PatientGeneratedData
+    module FhirClient
```

Comment:
So funnily enough, FHIR does have an inflection in the inflections file, so this should be `FHIRClient`

Reply:
Good catch! Changing now!

Reply:
Done!